### ISSUE TITLE
Timer for writing cards and voting

### ISSUE THREAD

**Issue by glauberramos** [funretro/distributed#91]
It could have a timer for the whole board. Everyone should be able to increase or decrease time and stop and start the timer.

Voting and creating cards should only be available while timer is running.

Once timer is over the cards are updated and everyone see the result.

It's like a automated facilitator.

**Comment by glauberramos** [comment]
In order to keep funretro simple I don't think this fits well in the scope.

**Comment by Roshanjossey** [comment]
@glauberramos: I think this is a great feature. We timebox almost everything in our retros.
Starting from writing issues, reading all issues, voting, discussions and decisions.

Currently we use our phones or google for timer. If we can put it in board itself, that could be the selling point for funretro against other tools.

Lemme see if I can get any designer friends involved in this.

**Comment by alanjsph** [comment]
@Roshanjossey , so the changes has to be reflected only after the timer stops or, just simply block the action of creation of cards and voting when the timer is expired?

**Comment by Roshanjossey** [comment]
Hey @alanjsph: In my opinion timer doesn't have to restrict creation and voting of cards.

It can just stay there as an indicator. (This is one aspect where I disagree with @glauberramos' comment above)

Anyways, get started with visual part of visual part of it. We'll continue our discussions with @glauberramos also involved.

**Comment by alanjsph** [comment]
Alright @Roshanjossey

**Comment by alanjsph** [comment]
![screen shot 2017-04-01 at 6 06 25 pm](https://cloud.githubusercontent.com/assets/8162618/24578845/2c70ba74-1707-11e7-8c31-4a376a6e0bce.png)
How does this look? @Roshanjossey

**Comment by glauberramos** [comment]
@Roshanjossey @alanjsph I think we should give a little bit more thought into this Idea. That's why I added a "Idea" label, because I was not sure how this feature should be implemented.

- I think in order to work it should be really simple, I agree with you @Roshanjossey that we should not limit the creation of cards.
- I don't think we should add on the header. Maybe we could have it floating on the bottom right of the page, and with simpler colors, like only grays and white. 
- I think it would be nice to don't show it initially, only if you enable the feature. 
- We could add a board configuration sidebar, or something like that, and on that place we should have the option to enable the timer. I took this inspiration from Trello:

<img width="140" alt="screen shot 2017-04-03 at 04 45 58" src="https://cloud.githubusercontent.com/assets/559434/24599894/a6a9ed34-1828-11e7-9d29-b56b6bd69545.png">

<img width="457" alt="screen shot 2017-04-03 at 04 46 36" src="https://cloud.githubusercontent.com/assets/559434/24599905/b204bda8-1828-11e7-993c-0dad2ab1355c.png">

Can we work more on the design right now and don't spend time developing it? I think this feature is important to have the design right, so we should focus on that first.
